sdk/identity/identity-vscode/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 1.0.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.0.1 (2025-04-08)
 
 ### Other Changes
 
+- Update Readme with the deprecation notice.
```

Comment:
Should we add something like - "This package is deprecated. See Readme for more details" ?

Reply:
Would that be in the Other Changes section or just in the beginning of the file in the new Changelog section?

Reply:
The "Other Changes" section makes the most sense to me. On a related note, is someone planning to mark the package as deprecated on npmjs.com? https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions#using-the-website

Reply:
Yes we are planning to mark it as deprecated on npm with this release

Reply:
Updated!